src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Syntax/Names.cs

```diff
@@ -4,23 +4,15 @@
 
 namespace Microsoft.CSharp.RuntimeBinder.Syntax
 {
-    internal class Name
+    internal sealed class Name
```

Comment:
Yes. This seems like a transparent wrapper of System.String which can/should be removed, but not in this change.

---
In reply to: [108411036](https://github.com/dotnet/corefx/pull/17605#discussion_r108411036) [](ancestors = 108411036)